lib/sellers/views.py

```diff
@@ -13,5 +13,6 @@ class SellerProductViewSet(NonDeleteModelViewSet):
     queryset = SellerProduct.objects.all()
     serializer_class = SellerProductSerializer
     filter_fields = (
-        'external_id', 'public_id', 'seller__uuid', 'seller__active'
+        'external_id', 'public_id', 'seller__uuid', 'seller__active',
+        'seller'
```

Comment:
Kind of satisfying that writing this code found this one missing, but we'll need to check flows on -dev to see if there's anything webpay or zamboni hits that we haven't tested for.